frontend/packages/dev-console/src/components/catalog/catalog-plugin.ts

```diff
@@ -0,0 +1,82 @@
+import { CatalogItemProvider, CatalogItemType, Plugin } from '@console/plugin-sdk';
+import { builderImageProvider, helmChartProvider, templateProvider } from './providers';
+
+export type CatalogConsumedExtensions = CatalogItemProvider | CatalogItemType;
+
+export const catalogPlugin: Plugin<CatalogConsumedExtensions> = [
+  {
+    type: 'Catalog/ItemType',
+    properties: {
+      type: 'BuilderImage',
+      title: 'Builder Images',
+      catalog: 'developer',
+      catalogDescription:
+        'Browse for container images that support a particular language or framework. Cluster administrators can customize the content made available in the catalog.',
+      typeDescription:
+        'Builder images are container images that build source code for a  particular language or framework.',
```

Comment:
UX had the type string as bold. Do we _need_ to bold it? Which means supporting react components here instead of strings. I'm happier to keep it as a string.

Reply:
I would like to keep it a string. We can talk to UX about this. If needed we can make it a markdown instead which will still be string type.